Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org/' do
 
 group :test do
   git 'https://github.com/Azure/terramodtest.git' do
-    gem 'terramodtest', tag: '0.5.0'
+    gem 'terramodtest', tag: '0.6.0'

```

Comment:
may I ask what's the meaning here? it's the tag of "terramodtest" repo? from here https://github.com/Azure/terramodtest/tags I could only find the tag "0.4.0"

Reply:
Right. It doesn't need to be released in repo. You can refer it directly in ruby.

Reply:
It's for terraform fmt check.